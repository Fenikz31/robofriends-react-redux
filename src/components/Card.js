import React from 'react';

const Card = ({ name, email, id }) => {
    return (
        <div className='tc bg-light-blue dib br3 pa3 ma2 grow shadow5'>
            <img src={`https://robohash.org/${id}?200x200`} alt='robots'/>
            <div>
                <h2>{name}</h2>
                <p>{email}</p>
            </div>
        </div>
    );
}

export default Card;
