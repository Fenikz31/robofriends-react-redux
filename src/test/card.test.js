import { shallow } from 'enzyme'
import React from 'react'
import Card from '../components/Card'

const wrapper = shallow(<Card />)

// console.log(wrapper)

it('expects to render Card component', () => {
    expect(wrapper.length).toMatchSnapshot()
})