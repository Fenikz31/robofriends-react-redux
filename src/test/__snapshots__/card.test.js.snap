// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render Card component 1`] = `1`;
